spring-aot/src/main/java/org/springframework/nativex/type/Field.java

```diff
@@ -64,7 +65,18 @@ public Field(FieldNode node, TypeSystem typeSystem) {
 		}
 		return results == null ? Collections.emptyList() : results;
 	}
-	
+
+	/**
+	 * @return {@literal true} if considered a final field (ACC_FINAL || LDC).
+	 */
+	public boolean isFinal() {
+		return Opcodes.ACC_FINAL == node.access || Opcodes.LDC == node.access;
```

Comment:
Is this quite right?  If I had a 'final static' field (`0x18`) this won't match? I also find it a little cryptic that you are using LDC as a shorthand for 'Opcodes.FINAL | Opcodes.PRIVATE' - I don't think I like that. It confuses the LDC opcode with the access flag values. Why isn't it written a bit more like the isSynthetic() one and testing for the final bit? `return (node.access & Opcodes.ACC_FINAL) != 0`.

Reply:
thanks - I'll change as you suggested.